### ISSUE TITLE
use the network architecture given by the paper

### ISSUE THREAD

**Issue by lintongtong123** [musicbeer/Deep-Learning-for-the-Physical-Layer#1]
Hello, I run your code but ind that the performance is far from the paper. Why don't you use the network architecture given by the paper, like the relu and softmax function?

**Comment by Zhouqinglin2020** [comment]
It seems that he have used the softmax function in nn.CrossEntropyLoss()
